### PR TITLE
Add removed integrations back

### PR DESCRIPTION
**Pull request by ollipa**
Add removed integrations back. I checked all integrations and the related projects seem to be active. `unittest2` is not maintained anymore so I didn't add that back.

I also updated authors in pyproject.toml and license. The license file seemed to have some small inconsistencies with the real BSD2 license (eg. "COPYRIGHT OWNER" -> "COPYRIGHT HOLDER") so I copied the license text from https://choosealicense.com/licenses/bsd-2-clause/.

### CODE REVIEW

File: flexmock/integrations.py [hunk 1]
```diff
@@ -57,11 +61,77 @@ def run(
...
+    # only valid TestResult class for unittest is TextTestResult
+    _patch_test_result(unittest.TextTestResult)
+
+
+def hook_into_teamcity_unittest() -> None:
+    """Hook into Teamcity unittests. This allows flexmock to be used within PyCharm."""
+    try:
+        from tcunittest import TeamcityTestResult
+    except ImportError:
+        pass
```
Comment (christophe-riolo) [inline flexmock/integrations.py:73]: Should we log instead of passing ?

File: pyproject.toml [hunk 2]
```diff
@@ -2,7 +2,11 @@
 name = "flexmock"
 version = "0.10.4"
 description = "flexmock is a testing library for Python that makes it easy to create mocks, stubs and fakes."
-authors = ["Your Name <you@example.com>"]
+authors = [
```
Comment (christophe-riolo) [inline pyproject.toml:5]: So did you check the difference between authors and maintainers ? Can we just replace the authors ?

**Review by christophe-riolo — COMMENTED** [review]
I'm fine with these changes, just small comments

File: pyproject.toml (hunk 2 above)
Comment (ollipa) [reply pyproject.toml]: Seems that authors indeed refers to the original authors:

https://www.python.org/dev/peps/pep-0345/#maintainer-optional
> Note that this field is intended for use when a project is being maintained by someone other than the original author: it should be omitted if it is identical to Author.

Fixup: https://github.com/flexmock/flexmock/pull/1/commits/197baea5d2fab0c3817df635b000db7891c57a92

File: flexmock/integrations.py (hunk 1 above)
Comment (ollipa) [reply flexmock/integrations.py:73]: I think it would unnecessary because we are expecting most of these imports to fail (if not all)

**Review by christophe-riolo — APPROVED** [review]
Looks good! 😃

**Review by adarshk7 — APPROVED** [review]
Looks good 👍

**Pull request merged**